.gitignore

```diff
@@ -77,8 +77,6 @@ local.properties
 *.log
 
 # Build results
-[Dd]ebug/
-[Rr]elease/
```

Comment:
Done in https://github.com/mantidproject/mantid/commit/87326e78f950ec70229afcc4dca74db3669771d8